src/cmd_line/commands/file.ts

```diff
@@ -23,13 +29,48 @@ export class FileCommand extends node.CommandBase {
         return this._arguments;
     }
 
+    getViewColumn(sideBySide: boolean) : vscode.ViewColumn {
+        const active = vscode.window.activeTextEditor;
+        if (!active) {
+            return vscode.ViewColumn.One;
+        }
+
+        if (!sideBySide) {
+            return active.viewColumn;
+        }
+
+        switch (active.viewColumn) {
+            case vscode.ViewColumn.One:
```

Comment:
So what is this function doing? What is the sideBySide parameter? This switch appears to get the view column one to the right, but how come it doesn't handle the ViewColumn.Three case? 

It seems to me like you might want two separate functions. getActiveViewColumn() and getViewColumnToRight()